Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -211,10 +209,19 @@ final class ManagePledgeViewController: UIViewController, MessageBannerViewContr
         self?.rewardReceivedViewController.configureWith(project: project)
       }
 
-    self.viewModel.outputs.configureRewardSummaryView
+    self.viewModel.outputs.loadProjectAndRewardsIntoDataSource
+      .observeForUI()
+      .observeValues { [weak self] project, rewards in
+        self?.dataSource.load(project: project, rewards: rewards)
+        self?.configureHeaderView()
```

Comment:
Does it not work to configure the header view in `viewDidLoad`?

Reply:
Oh I see, it's to support the pull to refresh behavior. Any reason why we can't just set the refresh control on the table view itself? It might not be ideal to be recreating all the table view header subviews each time the data is reloaded 🤔